content/player/blockchain-integration/buying-nfts.md

```diff
@@ -39,11 +39,11 @@ This is a list of smart contracts accessible to users through the marketplace ap
 
 As you can see, there are contracts both in the Ethereum and the Polygon networks. This is because different kinds of assets can be found in one network and the rest in the other. For example, LANDs, Estates, and Names are all in the Ethereum Network, while Wearables and Emotes are in the Polygon Network.
 
-**Networks are not interoperable, meaning that you cannot buy Wearables in the Ethereum network or place bids for LANDs in Polygon. Contracts can only interact with the assets found in their same networks.**
+**Networks are not interoperable, meaning that you cannot buy Wearables in the Ethereum network or place bids for LANDs in Polygon. Contracts can only interact with the assets found in their same networks. But still, the user can get Wearables or Emotes while connected to the Ethereum network thanks to [Meta-Transactions](https://docs.decentraland.org/player/blockchain-integration/transactions-in-polygon/), this is automatic in the Marketplace for a better UX**
 
 ## Marketplace
 
-The Marketplace Smart Contract is the one that allows users to create an Order to sell their assets. On Ethereum, you can create an Order to sell LANDs, Estates, Names and Wearables (Legacy) while on Polygon, the Marketplace Smart Contract allows the user to create Orders for Wearables, Emotes, etc.
+The Marketplace Smart Contract is the one that allows users to create an Order to sell their assets.
```

Comment:
If this is a technical document (specially because there's solidity code here), I wouldn't remove this explanation as it explains what can be done in each network. We can enhance it though:
```suggestion
The Marketplace Smart Contract in Ethereum allows users to create an Order to sell LANDs, Estates, Names and Legacy Wearables, while the one in Polygon only allows the creation of Orders for Wearables and Emotes.
```